attack.go

```diff
@@ -50,6 +50,7 @@ func attackCmd() command {
 	fs.Var(&opts.headers, "header", "Request header")
 	fs.Var(&opts.laddr, "laddr", "Local IP address")
 	fs.BoolVar(&opts.keepalive, "keepalive", true, "Use persistent connections")
+	fs.StringVar(&opts.unixSocket, "unix-socket", "", "Connect over a unix socket")
```

Comment:
This change should be reflected in the README.

Reply:
We should also clarify that the host part of the target URLs are ignored when using UNIX domain sockets.

Reply:
added to flag  doc and updated readme